### PR TITLE
Add debian packaging to goreleaser

### PR DESCRIPTION
**Pull request by antekresic**

### CODE REVIEW

File: build/.goreleaser.yml [hunk 1]
```diff
@@ -33,6 +27,54 @@ scoop:
...
+
+    license: Apache License 2.0
+    maintainer: Timescale <info@timescale.com>
+    contents:
+      - src: build/systemd/promscale.service
+        dst: /etc/systemd/system/promscale.service
+      - src: build/conf/promscale.conf
+        dst: /etc/sysconfig/promscale.conf
+        type: "config|noreplace"
+        packager: rpm
```
Comment (cevian) [inline build/.goreleaser.yml:61]: how does this conf file work on deb then?

File: build/scripts/postinstall.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+echo "Promscale installed as a systemd service"
+echo "Enable auto-start with: systemctl enable promscale"
+echo "And start the service now with: systemctl start promscale"
+echo "----------------------------------------"
+CONF_FILE=$(find /etc -name promscale.conf 2>/dev/null | head -n 1)
```
Comment (cevian) [inline build/scripts/postinstall.sh:6]: why don't we know where this is? How does this work on deb?

**Review by cevian — CHANGES_REQUESTED** [review]
This looks great. some questions about deb packaging. Also was this all manually tested on rhel, debian, and ubuntu?

File: build/scripts/postinstall.sh (hunk 2 above)
Comment (antekresic) [reply build/scripts/postinstall.sh:6]: Config files go into different places in `/etc` folder. I did miss to add the location for deb package, will add.

File: build/.goreleaser.yml (hunk 1 above)
Comment (antekresic) [reply build/.goreleaser.yml:61]: I somehow missed adding this conf file for deb packaged, will add now.

**Comment by antekresic** [comment]
Updated to add missing deb conf file location. Manually tested with `yum` and `dpkg` package installers.

**Review by cevian — APPROVED** [review]

**Pull request merged**